### PR TITLE
TRUNK-3343: insert the new application level privileges for each each…

### PR DESCRIPTION
**Pull request by mozzy11**
TICKET :https://issues.openmrs.org/browse/TRUNK-3343

### CODE REVIEW

File: api/src/main/resources/liquibase-update-to-latest.xml [hunk 1]
```diff
@@ -21,4 +21,73 @@
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
 
+
+<changeSet id="TRUNK-3343-assigning-new-app-privileges" author="mozzy">
+
```
Comment (dkayiwa) [inline api/src/main/resources/liquibase-update-to-latest.xml]: Shouldn't you add some preconditions?

Comment (mozzy11) [reply api/src/main/resources/liquibase-update-to-latest.xml]: oh ya but in this case i failed to figure out an appropriate pre condition. May be u can give me a hint

Comment (mozzy11) [reply api/src/main/resources/liquibase-update-to-latest.xml]: i added that pre-condition

**Comment by ibacher** [comment]
This probably needs to be refactored to take into account our new Liquibase structure.

**Pull request closed without merging**